graphql/queries/dashboardQuery.graphql

```diff
@@ -1,12 +1,12 @@
 query getDashboardPage {
-	scLabsErrorPageByPath(_path: "/content/dam/decd-endc/content-fragments/alpha/sclabs/pages/500-error-page") {
-	    item {
-            sclContentEn {
-                json
-            }
-            sclContentFr {
-                json
-            }
-        }
-	}
-}
\ No newline at end of file
+  alphaSCHPageByPath(
+    _path: "/content/dam/decd-endc/content-fragments/alpha/secure-client-hub/pages/dashboard"
+  ) {
+    item {
+      scPageNameEn
+      scPageNameFr
+      scTitleEn
+      scTitleFr
```

Comment:
hmm, I don't think this is an ideal formatting...
This requires us to build/maintain that mapping function (`MapDashboard`) just so we can toggle the language - that's going to be very error prone.
A preferable method, in my opinion, is one where we don't need to "map" the objects returned from AEM. While we don't control objects from Interop, we should from AEM. We can edit the application to interpret the new AEM object (rather than the current one), but it needs to enable us to toggle language.

Again, any change here would require AEM change too, but I see this a bit more as a "blocker".

Reply:
This is absolutely critical and @krischarbonneau  with SCLabs is planning to follow suit. We have found in the past that this is less error prone than trusting the CMS response

Reply:
As agreed in the meeting, we will keep the mapping.